docs/RulesOverview.md

```diff
@@ -1,4 +1,7 @@
-## Rules Overview
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Not changed by this PR, but noticed that at line 23 in this file, Inheritence should be spelled Inheritance